backend/src/apiserver/resource/resource_manager.go

```diff
@@ -218,6 +218,8 @@ func (r *ResourceManager) CreateRun(apiRun *api.Run) (*model.RunDetail, error) {
 		for artIdx, artifact := range template.Outputs.Artifacts {
 			if artifact.Name == "mlpipeline-ui-metadata" || artifact.Name == "mlpipeline-metrics" {
 				workflow.Workflow.Spec.Templates[templateIdx].Outputs.Artifacts[artIdx].Optional = true
+			} else {
+				artifact.Archive = &workflowapi.ArchiveStrategy{None: &workflowapi.NoneStrategy{}}
```

Comment:
nit: why do we add this in an else clause here?
I think we can add it to all.

Reply:
Another issue is that, there should be some similar code in `CreateJob` method, you should add this part there too

Reply:
>nit: why do we add this in an else clause here?

I'm not sure that Frontend and Backend Metrics are prepared for uncompressed artifacts.

Reply:
Frontend server should support both compressed and uncompressed artifacts. 

https://github.com/kubeflow/pipelines/blob/4e5d724fef23d504ff1be02191c0f992672dff37/frontend/server/minio-helper.ts#L134